### PR TITLE
fix: links for nft type to indicate metadata and add toggle for webview

### PR DESCRIPTION
**Pull request by cmunns**
Updates the links title,text,url if nft type but also bridges a gap I feel is needed. In the event that a user is accessing the webapp but doesn't have IPFS downloaded we should provide a toggle to use the webviewer.

Open to other ideas, but for now I think is a good stop gap to prevent broken links for new users.

Screen Shot 2022-02-04 at 2 31 41 PM

### CODE REVIEW

File: packages/website/pages/files.js [hunk 1]
```diff
@@ -302,7 +321,19 @@ export default function Files({ user }) {
...
+                <div className="flex-auto chicagoflf mv3">
+                  <h1>Files</h1>
+                  <label className="webViewer-toggle mt1">
+                    <input
+                      type="checkbox"
+                      name="toggleWebViewer"
+                      className="mr1"
+                      onChange={(e) => handleWebViewerToggle(e)}
+                    />
+                    Use IPFS web viewer instead of IPFS applications where
```
Comment (alanshaw) [inline packages/website/pages/files.js:333]: I'm not sure what "web viewer" means!? It means open on the gateway in your browser?

**Review by alanshaw — DISMISSED** [review]

File: packages/website/pages/files.js (hunk 1 above)
Comment (cmunns) [reply packages/website/pages/files.js:333]: Yes, not sure on the correct wording here. Open to your input...just felt important to have an opt-in for users that want to use https://ipfs.io as the viewer.

**Comment by Codigo-Fuentes** [comment]
URL-fix

File: packages/website/pages/files.js (hunk 1 above)
Comment (JeffLowe) [reply packages/website/pages/files.js:333]: @alanshaw is the concern the label used, or the underlying approach of giving user the choice?

**Review by dashcraft — APPROVED** [review]
LGTM

comment (non blocking): this is one of the files I've flagged as needing a refactor in the future

**Pull request merged**